src/CricketGame.java

```diff
@@ -0,0 +1,41 @@
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+
+
+public class CricketGame {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        Team India = new Team("India");
+        Team Pakistan = new Team("Pakistan");
+        int IndiaWins = 0;
```

Comment:
please take care of the camel-case naming styles, `IndiaWins` can be `indiaWins`